File: javascript/nivel 2 logica/10.js
/*
Ejercicio 10
Dividir dos números sin usar el operador /
(división entera)
*/

let dividendo = parseInt(prompt("Ingrese el Dividendo: "));
let divisor = parseInt(prompt("Ingrese el Divisor: "));

if(divisor === 0){
  console.log("Error, no se puede dividir por 0");
}else{
  let resultado = 0;

  while(dividendo >= divisor){
    dividendo -= divisor;
    resultado++;
  }
  console.log("El resultado de la division es: ", resultado);
}

File: python/nivel 2 logica/10.py
"""
Ejercicio 10
Dividir dos números sin usar el operador /
(division entera)
"""

dividendo = int(input("Ingrese el dividendo: "))
divisor = int(input("Ingrese el divisor: "))

if divisor == 0:
  print("Error: no se puede dividir por cero")
else:
  resultado = 0

  while dividendo >= divisor:
    dividendo -= divisor
    resultado +=1

print(f"El resultaod de la division es: {resultado}")
